analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -263,20 +285,73 @@ cnv_no_xy <- cnv_df %>%
 
 # Merge and annotated no X&Y
 autosome_annotated_cn <- process_annotate_overlaps(cnv_df = cnv_no_xy,
-                                                   txdb_exons = tx_exons) %>%
+                                                   txdb_exons = exons_gr) %>%
```

Comment:
Probably should change the name of this argument to reflect that it isn't that specific txdb object anymore.